src/main/java/com/salesforce/mirus/config/SourceConfig.java

```diff
@@ -8,20 +8,19 @@
 
 package com.salesforce.mirus.config;
 
+import static java.util.stream.Collectors.toList;
+
```

Comment:
Are these formatting changes necessary?  Remove if not.

Reply:
I think it's necessary, they were formatted when I run maven command.